src/main/java/mx/org/corresponsabilidadsocial/api/blog/controllers/PublicacionController.java

```diff
@@ -0,0 +1,27 @@
+package mx.org.corresponsabilidadsocial.api.blog.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+import mx.org.corresponsabilidadsocial.api.blog.services.PublicacionService;
+
+@RestController
+@RequestMapping("/blog")
+public class PublicacionController {
+
+    @Autowired
+    PublicacionService publicacionService;
+
+    @GetMapping("/posts")
+    public ResponseEntity<?> traerListaPublicaciones() throws Exception{
+
+        return new ResponseEntity<>(publicacionService.traerListaPublicaciones(), HttpStatus.OK);
```

Comment:
Habria que agregar la logica de validar si la lista de publicaciones es vacia, retornar un NOT_FOUND

Reply:
listo, retorna un not found en caso de que la lista este vacía, pero no como excepción.